tests/test_usage.py

```
@@ -661,6 +661,33 @@ def test_prints_version_WHEN_called_GIVEN_version_argument(self):
 				theResult = False
 		self.assertTrue(theResult, str("""Could Not find version from multicast --version"""))
 
+	def _validate_help_output(self, args):
```

Ruff (0.8.2)

Missing return type annotation for private function `_validate_help_output`

(ANN202)